src/docs/asciidoc/data-access.adoc

```diff
@@ -5567,16 +5558,14 @@ declared type `ITEM_TYPE`:
 		public TestItemStoredProcedure(DataSource dataSource) {
 			// ...
 			declareParameter(new SqlOutParameter("item", OracleTypes.STRUCT, "ITEM_TYPE",
-				new SqlReturnType() {
-					public Object getTypeValue(CallableStatement cs, int colIndx, int sqlType, String 	) throws SQLException {
```

Comment:
missing param name in `String typeName`

Reply:
That's true, but this is in the documentation that is being replaced. Thus we can ignore that for now.